app/services/search_service.rb

```diff
@@ -84,6 +84,13 @@ def search_tags(query, limit = 10)
       .includes(:node)
       .references(:node)
       .where('node.status = 1')
+      .order('
+        CASE
```

Comment:
Hey!
Just a quick suggestion can we store the `case` string in a variable and then just use that inside order?
For example: tags_order = "CASE WHEN name like #{query} THEN 1 WHEN name LIKE #{query}% THEN 2 WHEN name LIKE %{query} THEN 4 ELSE 3"
and then just use `order(tags_order)`.

Also, should we include one more case: `WHEN name like %#{query}% `

Reply:
@namangupta01, the first suggestion seems to be a good idea to store the string as  a variable and using it. I'm adding a commit with the changes. 
As per `WHEN name like %#{query}%` case is concerned, I think it will show up in the ELSE condition as it is the only condition left for else ig. What do you think?

Reply:
Hi @namangupta01, kindly give this another look when you get some time. Thanks and nice to you see around!